missouri/demographics.yml

```diff
@@ -27,10 +27,10 @@ demographics:
                 init: 0.496
             haart:
               init: 0.708
-              enroll: # unsure of start and stop times
+              enroll:
                 enroll_1:
                   start: 0
-                  stop: 1
+                  stop: 2
```

Comment:
should enroll_2 stop at 13 instead of 12?

Reply:
oh yes youre right